airbyte-integrations/connectors/source-snowflake/source_snowflake/schema_builder.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict
+
+
+class SchemaTypes:
```

Comment:
It seems like the float type is not working. I have a table defined as:
```
create or replace TABLE ALEX_TEST_DB.POSTHOG.ADACCOUNTS_STATS_DAILY cluster by (_AIRBYTE_UNIQUE_KEY, _AIRBYTE_EMITTED_AT)(
	_AIRBYTE_UNIQUE_KEY VARCHAR(32),
	ID VARCHAR(16777216),
	TYPE VARCHAR(16777216),
	SAVES FLOAT,
<...>
);
```

The discovered catalog has `null` has in the schema for the floats:
```
                "json_schema": {
                    "$schema": "https://json-schema.org/draft-07/schema#",
                    "type": "object",
                    "additionalProperties": true,
                    "properties": {
                        "_AIRBYTE_UNIQUE_KEY": {
                            "type": [
                                "null",
                                "string"
                            ]
                        },
                        "ID": {
                            "type": [
                                "null",
                                "string"
                            ]
                        },
                        "TYPE": {
                            "type": [
                                "null",
                                "string"
                            ]
                        },
                        "SAVES": null,
                        <...>
                },
```

Reply:
I made a mistake concerning the mapping I have put the integer twice instead of float. I will correct that